src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -63,7 +52,9 @@ public function elementValidate($element, FormStateInterface $form_state, $form)
       return;
     }
 
-    $entity = entity_load($this->getFieldSetting('target_type'), $matches[1]);
+    $entity = \Drupal::entityManager()
```

Comment:
There was a bug back then that if user will replace the entity ID with another entity ID it will be grouped even if the user don't have access to that group. In other words - we still need to validate it.